docs/anchoring-assets/anchor-platform/sep10/configuration.mdx

```diff
@@ -40,17 +40,43 @@ By default, the Anchor Platform allows anyone with a Stellar account to authenti
 ```bash
 # dev.env
 SEP10_CLIENT_ATTRIBUTION_REQUIRED=true
-SEP10_CLIENT_ATTRIBUTION_ALLOWLIST=lobstr.co,api.vibrantapp.com,decaf.so,api.beansapp.com
+```
+
+</CodeExample>
+
+<CodeExample>
 
-SEP10_REQUIRE_KNOWN_OMNIBUS_ACCOUNT=true
-SEP10_OMNIBUS_ACCOUNT_LIST=GBIBMZNXMD3P7HXVQCYIWWT5NG43NEIIY7VYBQ5SADV6UULUKCAJTGPG
+```yaml
+clients:
```

Comment:
Can you have 2 examples of custodial and non custodial separated a bit and add more comments to it (e.g. list fields that are required and optional for custodial/noncustodial)